proto/server/clientEvtHandler.go

```diff
@@ -94,29 +94,29 @@ type userPlaneProfile struct {
 }
 
 type qosProfile struct {
+	Ambr []int32 `json:"apn-ambr,omitempty"`
 	Qci  int32   `json:"qci,omitempty"`
 	Arp  int32   `json:"arp,omitempty"`
-	Ambr []int32 `json:"apn-ambr,omitempty"`
 }
 
 type configSpgw struct {
-	SubSelectRules    []*subSelectionRule          `json:"subscriber-selection-rules,omitempty"`
 	ApnProfiles       map[string]*apnProfile       `json:"apn-profiles,omitempty"`
 	UserPlaneProfiles map[string]*userPlaneProfile `json:"user-plane-profiles,omitempty"`
 	QosProfiles       map[string]*qosProfile       `json:"qos-profiles,omitempty"`
+	SubSelectRules    []*subSelectionRule          `json:"subscriber-selection-rules,omitempty"`
 }
 
 type configHss struct {
+	Key         string                 `json:"Key,omitempty"`
+	ApnProfiles map[string]*apnProfile `json:"apn-profiles,omitempty"`
```

Comment:
Why not the following?
```suggestion
	ApnProfiles map[string]*apnProfile `json:"apn-profiles,omitempty"`
	Key         string                 `json:"Key,omitempty"`
```

Reply:
Same comment as above